src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/troubleshooting/troubleshoot-enabling-serverless-monitoring-aws-lambda.mdx

```diff
@@ -29,7 +29,7 @@ Besides these basic enablement problems, there are some additional problems that
 * You've completed the installation, integration, and instrumentation steps correctly, and your function is sending logs to CloudWatch but you're not seeing traces for specific dependencies (or any traces) in the UI. This may result from the order of layer merging (if you're using our Lambda layers) or from the order of import (if you're instrumenting manually):
 
   * If you're instrumenting with layers: make sure in your function configuration that the New Relic layer is merged before other layers (though if your function uses webpack, the New Relic layer should be merged after the webpack layer).
-  * If you're instrumenting a Node function manually, make sure that your function imports **newrelic** and **@newrelic/aws-sdk** before it imports any dependencies you expect to monitor.
+  * If you're instrumenting a Node function manually, make sure that [logging is enabled](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#logging_config), and that your function imports **newrelic** and **@newrelic/aws-sdk** before it imports any dependencies you expect to monitor.
```

Comment:
The nodejs agent already imports `@newrelic/aws-sdk` as a dependency.
https://github.com/newrelic/node-newrelic/blob/781ae9eb433645f658aa0b5673b4dffb12e5b0de/lib/instrumentations.js#L13

Reply:
Thanks, I've removed it.